packages/react-async/src/reducer.js

```diff
@@ -1,5 +1,7 @@
 import { getInitialStatus, getIdleStatus, getStatusProps, statusTypes } from "./status"
 
+export const neverSettle = new Promise(() => {})
```

Comment:
If we expose this to the user, this might be a memory leak.
I think it should be something chainable with a promise-like interface that never holds a reference to the argument it is being passed, like
```typescript
export const neverSettle = {
    finally() { 
        return this;
    },
    catch() { 
        return this;
    },
    then() { 
        return this;
    }
}
```

Reply:
👍